package_schemas/country_estimates/2_country_estimates_22.json

```diff
@@ -0,0 +1,356 @@
+{
+    "scheming_version": 1,
+    "dataset_type": "country-estimates-22",
+    "name": "Country Estimates 2022",
+    "link_name": "Country Estimates 2022",
+    "about": "",
+    "about_url": "http://github.com/ckan/ckanext-scheming",
+    "add_data_on_create": false,
+    "dataset_fields": [{
+            "field_name": "title",
+            "label": "Title (autogenerated):",
+            "form_snippet": "text.html",
+            "validators": "autogenerate",
+            "classes": ["hidden"],
+            "template": "{} Inputs UNAIDS Estimates 2022",
+            "template_args": ["geo-location", "year"],
+            "template_formatters": ["comma_swap"],
+            "display_group": "Country Estimates 2022"
+        },
+        {
+            "field_name": "name",
+            "label": "URL",
+            "form_snippet": "text.html",
+            "validators": "autogenerate auto_create_valid_name not_empty unicode name_validator package_name_validator",
+            "classes": ["hidden"],
+            "template": "{}-inputs-unaids-estimates-2022",
```

Comment:
shouldn't we get read of "UNADIS Estimates" from files & variables names all together?

Reply:
Yeah excellent catch. Thankyou.